src/datumaro/plugins/data_formats/datumaro/exporter.py

```diff
@@ -179,6 +179,24 @@ def context_save_media(
         """
         if item.media is None:
             yield
+        elif isinstance(item.media, Video):
+            video = item.media_as(Video)
+
+            if context.save_media:
+                fname = context.make_video_filename(item)
+                # To prevent the video from being overwritten
+                # (A video can have same path but different start/end frames)
+                if not osp.exists(fname):
+                    context.save_video(item, fname=fname, subdir=item.subset)
+                item.media = Video(
+                    path=video.path,
+                    step=video._step,
+                    start_frame=video._start_frame,
+                    end_frame=video._end_frame,
+                )
+
+            yield
+            item.media = video
```

Comment:
I have a question. A video data is clipped into multiple files when exporting. Is this correct?
I think it would be better to save start/end frame indices in annotation file when exporting with Datumaro format. How about you?

Reply:
start/end frame indices are saved in the item description in the annotation file, like VideoFrame . 
This is an example annotation file.
```
{
    "dm_format_version":"1.0",
    "media_type":1,
    "infos":{},
    "categories":{ ... },
    "items":[
        {
            "id":"f1",
            "annotations":[ ... ],
            "video_frame": {"video_path":"video.avi","frame_index":0}
        },
        {
            "id":"v1",
            "annotations":[ ... ],
            "video": {
                "path":"/path/to/video.avi",
                "step":1,
                "start_frame":2,
                "end_frame":2
            }
        }
    ]
}
```